doc/manual/dates.rst

```diff
@@ -73,6 +73,13 @@ One note on parsing performance: using the ``Date(date_string,format_string)`` f
   2015-01-02
 
 
+To print a :class:`Date` or :class:`DateTime` object as a string use the ``Dates.format`` function:
+
+::
+
+  julia> Dates.format(Date(2016,2,1), "uuu d, yyyy")
```

Comment:
The date format strings really deserve to be in this rst too (rather than in the Datetime docstring).

Aside: this is pretty weird behavior (I found the repeated `u` in your example interesting):
```
julia> Dates.format(Date(2016,2,1), "uuu d, yyyy")
"Feb 1, 2016"

julia> Dates.format(Date(2016,2,1), "u d, yyyy")
"Feb 1, 2016"

julia> Dates.format(Date(2016,2,1), "u d, y")
"Feb 1, 6"
```